cms/djangoapps/contentstore/management/commands/tests/test_create_course.py

```diff
@@ -61,3 +61,35 @@ def test_all_stores_user_email(self, store):
         )
         # pylint: disable=protected-access
         self.assertEqual(store, modulestore()._get_modulestore_for_courselike(new_key).get_modulestore_type())
+
+    @ddt.data(ModuleStoreEnum.Type.split, ModuleStoreEnum.Type.mongo)
+    def test_get_course_with_different_case(self, default_store):
+        """
+        Tests that course can not be accessed with different case.
+
+        Scenario:
+            Create a course with lower case keys inside `bulk_operations` with `ignore_case=True`.
+            Verify that course is created.
+            Verify and that get course from store using same course id but different case is not accessible.
+        """
+        org = 'org1'
+        number = 'course1'
+        run = 'run1'
+        with self.store.default_store(default_store):
+            lowercase_course_id = self.store.make_course_key(org, number, run)
+            with self.store.bulk_operations(lowercase_course_id, ignore_case=True):
+                # Create course with lowercase key & Verify that store returns course.
+                self.store.create_course(
+                    lowercase_course_id.org,
+                    lowercase_course_id.course,
+                    lowercase_course_id.run,
+                    self.user.id
+                )
+                course = self.store.get_course(lowercase_course_id)
+                self.assertIsNotNone(course, 'Course not found using lowercase course key.')
+                self.assertEqual(unicode(course.id), unicode(lowercase_course_id))
+
+                # Verify store does not return course with different case.
+                uppercase_course_id = self.store.make_course_key(org.upper(), number.upper(), run.upper())
+                course = self.store.get_course(uppercase_course_id)
```

Comment:
@mushtaqak  If your questions is, That we will not access course with different case?
Well, this also the current behavior that we can not access courses with different cases.
If a course key is `a/b/c/` it will not be accessed with `A/B/C/` this is current behavior.

This test is to justify that this behavior still holds true, even with the change in  `store.bulk_operations` using `ignore_case=True`.